server/src/main/kotlin/suwayomi/tachidesk/graphql/mutations/ChapterMutation.kt

```diff
@@ -122,17 +124,21 @@ class ChapterMutation {
         val (clientMutationId, mangaId) = input
 
         return future {
+            val currentChapters = Chapter.getChapterList(input.mangaId)
```

Comment:
I think you could change this for a database query that just gets the chapter list size now that I think about it